### PR TITLE
[#21420] Keycard - Auth with password

### LINKED ISSUE #21420: Keycard - Auth with password
Image
 

Figma: https://www.figma.com/design/YGm3igIOAcwMqUVJWCJ6f1/Keycard-UX?node-id=469-70596&t=oxPq60BknspiPEJU-1

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #21420

QA : because there is no changes in the current code, this PR will be merged without e2e

### CODE REVIEW

**Review by mohsen-ghafouri — APPROVED** [review]

File: src/status_im/contexts/keycard/authorise/view.cljs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+       :background :blur
+       :icon-name  :i/close
+       :on-press   events-helper/navigate-back}]
+     [quo/page-top
+      {:title       (i18n/label :t/authorise-with-password)
+       :description :context-tag
+       :context-tag {:full-name           profile-name
+                     :profile-picture     profile-picture
+                     :customization-color customization-color}}]
+     [rn/view {:style {:flex 1 :padding-horizontal 20 :justify-content :space-between}}
```
Comment (clauxx) [inline src/status_im/contexts/keycard/authorise/view.cljs:27]: should be in the styles ns

File: src/status_im/contexts/keycard/authorise/view.cljs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      [quo/text (i18n/label :t/migrate-key-pair-authorise)]
+      [standard-auth/slide-button
+       {:size                :size-48
+        :container-style     {}
```
Comment (clauxx) [inline src/status_im/contexts/keycard/authorise/view.cljs:31]: not needed I guess

File: src/status_im/contexts/keycard/authorise/view.cljs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        :customization-color customization-color
+        :track-text          (i18n/label :t/slide-to-authorise)
+        :on-auth-success     #(println "TBD" (security/safe-unmask-data %))
```
Comment (clauxx) [inline src/status_im/contexts/keycard/authorise/view.cljs:34]: Is this screen functionality hidden behind a feature flag?

**Review by clauxx — APPROVED** [review]
Looks good 👍

File: src/status_im/contexts/keycard/authorise/view.cljs (hunk 2 above)
Comment (flexsurfer) [reply src/status_im/contexts/keycard/authorise/view.cljs:31]: it's needed , for some reason slide-button has a style with flex 1, which is strange

File: src/status_im/contexts/keycard/authorise/view.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/contexts/keycard/authorise/view.cljs:27]: not mandatory, here is fine

**Pull request merged**